template.go

```diff
@@ -68,6 +68,14 @@ func add(arg1, arg2 int) int {
 	return arg1 + arg2
 }
 
+func isTrue(bool string) bool {
+  b, err := strconv.ParseBool(strings.ToLower(bool))
```

Comment:
I wanted the function to return true regardless of the spelling, i.e. "TRUE", "tRuE", etc. should also return true. Does that make sense?